### PR TITLE
Namespace change from rosBridge -> ros

### PR DESCRIPTION
**Pull request by saching13**

### CODE REVIEW

File: depthai_bridge/include/depthai_bridge/BridgePublisher.hpp [hunk 1]
```diff
@@ -31,14 +27,23 @@ namespace rosOrigin = rclcpp;
...
+namespace ImageMsgs = sensor_msgs::msg;
+using ImagePtr = ImageMsgs::Image::SharedPtr;
+namespace rosOrigin = rclcpp;
+#else
 namespace StdMsgs = std_msgs;
 namespace ImageMsgs = sensor_msgs;
 using ImagePtr = ImageMsgs::ImagePtr;
-namespace rosOrigin = ros;
+namespace rosOrigin = ::ros;
 #endif
```
Comment (themarpe) [inline depthai_bridge/include/depthai_bridge/BridgePublisher.hpp:44]: Probably better to go one layer deeper into `dai::ros` just so core doesn't clash with this by any chance (low chances, still nicely separated.
You'll still be able to access `rosOrigin` as is now, in `dai::ros`. 

`rosBridge` is okay as it needs to be `dai::rosBridge`

File: depthai_bridge/include/depthai_bridge/DisparityConverter.hpp [hunk 2]
```diff
@@ -22,16 +20,22 @@ using DisparityImagePtr = DisparityMsgs::DisparityImage::SharedPtr;
...
+
+namespace dai {
+
+#ifdef IS_ROS2
+namespace DisparityMsgs = stereo_msgs::msg;
+using DisparityImagePtr = DisparityMsgs::DisparityImage::SharedPtr;
+#else
 namespace DisparityMsgs = stereo_msgs;
 using DisparityImagePtr = DisparityMsgs::DisparityImage::Ptr;
 #endif
```
Comment (themarpe) [inline depthai_bridge/include/depthai_bridge/DisparityConverter.hpp:33]: As above, into `dai::ros` namespacce

File: depthai_bridge/src/DisparityConverter.cpp [hunk 3]
```diff
@@ -1,7 +1,7 @@
 
 #include <depthai_bridge/DisparityConverter.hpp>
 
-namespace dai::rosBridge {
+namespace dai::ros {
```
Comment (themarpe) [inline depthai_bridge/src/DisparityConverter.cpp:4]: Modify to 2 namespaces

File: depthai_bridge/src/DisparityConverter.cpp [hunk 4]
```diff
@@ -42,7 +42,7 @@ void DisparityConverter::toRosMsg(std::shared_ptr<dai::ImgFrame> inData, Dispari
     outDispImageMsg.t = _baseline / 100;  // converting cm to meters
     sensor_msgs::msg::Image& outImageMsg = outDispImageMsg.image;
 #else
-    auto rosNow = ros::Time::now();
+    auto rosNow = ::ros::Time::now();
```
Comment (themarpe) [inline depthai_bridge/src/DisparityConverter.cpp:45]: Use `rosOrigin::Time::now()`?

File: depthai_bridge/src/ImageConverter.cpp [hunk 5]
```diff
@@ -5,7 +5,7 @@
 #include <ratio>
 #include <tuple>
 
-namespace dai::rosBridge {
+namespace dai::ros {
```
Comment (themarpe) [inline depthai_bridge/src/ImageConverter.cpp:8]: Modify to 2 namespaces

File: depthai_bridge/src/ImgDetectionConverter.cpp [hunk 6]
```diff
@@ -1,6 +1,6 @@
 #include <depthai_bridge/ImgDetectionConverter.hpp>
 
-namespace dai::rosBridge {
+namespace dai::ros {
```
Comment (themarpe) [inline depthai_bridge/src/ImgDetectionConverter.cpp:3]: Modify to 2 namespaces

File: depthai_bridge/src/ImgDetectionConverter.cpp [hunk 7]
```diff
@@ -15,7 +15,7 @@ void ImgDetectionConverter::toRosMsg(std::shared_ptr<dai::ImgDetections> inNetDa
 #ifndef IS_ROS2
     if(sequenceNum != -1) _sequenceNum = sequenceNum;
     opDetectionMsg.header.seq = _sequenceNum;
-    opDetectionMsg.header.stamp = ros::Time(sec, nsec);
+    opDetectionMsg.header.stamp = ::ros::Time(sec, nsec);
```
Comment (themarpe) [inline depthai_bridge/src/ImgDetectionConverter.cpp:18]: `rosOrigin::Time(..)`

File: depthai_bridge/src/ImgDetectionConverter.cpp [hunk 8]
```diff
@@ -28,7 +28,7 @@ void ImgDetectionConverter::toRosMsg(std::shared_ptr<dai::ImgDetections> inNetDa
 #ifndef IS_ROS2
     opDetectionMsg.header.seq = _sequenceNum;
     _sequenceNum++;
-    opDetectionMsg.header.stamp = ros::Time::now();
+    opDetectionMsg.header.stamp = ::ros::Time::now();
```
Comment (themarpe) [inline depthai_bridge/src/ImgDetectionConverter.cpp:31]: `rosOrigin::Time(..)`

File: depthai_bridge/src/SpatialDetectionConverter.cpp [hunk 9]
```diff
@@ -28,7 +28,7 @@ void SpatialDetectionConverter::toRosMsg(std::shared_ptr<dai::SpatialImgDetectio
 // setting the header
 #ifndef IS_ROS2
     opDetectionMsg.header.seq = _sequenceNum;
-    opDetectionMsg.header.stamp = ros::Time::now();
+    opDetectionMsg.header.stamp = ::ros::Time::now();
```
Comment (themarpe) [inline depthai_bridge/src/SpatialDetectionConverter.cpp:31]: `rosOrigin::Time(..)`

File: depthai_bridge/src/SpatialDetectionConverter.cpp [hunk 10]
```diff
@@ -16,7 +16,7 @@ void SpatialDetectionConverter::toRosMsg(std::shared_ptr<dai::SpatialImgDetectio
 #ifndef IS_ROS2
     if(sequenceNum != -1) _sequenceNum = sequenceNum;
     opDetectionMsg.header.seq = _sequenceNum;
-    opDetectionMsg.header.stamp = ros::Time(sec, nsec);
+    opDetectionMsg.header.stamp = ::ros::Time(sec, nsec);
```
Comment (themarpe) [inline depthai_bridge/src/SpatialDetectionConverter.cpp:19]: `rosOrigin::Time(..)`

File: depthai_bridge/src/SpatialDetectionConverter.cpp [hunk 11]
```diff
@@ -1,6 +1,6 @@
 #include <depthai_bridge/SpatialDetectionConverter.hpp>
 
-namespace dai::rosBridge {
+namespace dai::ros {
```
Comment (themarpe) [inline depthai_bridge/src/SpatialDetectionConverter.cpp:3]: Modify to 2 namespaces

File: depthai_bridge/src/ImuConverter.cpp [hunk 12]
```diff
@@ -1,15 +1,15 @@
 
 #include <depthai_bridge/ImuConverter.hpp>
 
-namespace dai::rosBridge {
+namespace dai::ros {
```
Comment (themarpe) [inline depthai_bridge/src/ImuConverter.cpp:4]: Modify to 2 namespaces

File: depthai_bridge/include/depthai_bridge/BridgePublisher.hpp [hunk 13]
```diff
@@ -31,14 +27,23 @@ namespace rosOrigin = rclcpp;
...
     #include "sensor_msgs/Image.h"
+#endif
+
+namespace dai {
+
+#ifdef IS_ROS2
+namespace StdMsgs = std_msgs::msg;
+namespace ImageMsgs = sensor_msgs::msg;
+using ImagePtr = ImageMsgs::Image::SharedPtr;
+namespace rosOrigin = rclcpp;
```
Comment (themarpe) [inline depthai_bridge/include/depthai_bridge/BridgePublisher.hpp:38]: `namespace rosOrigin = ::rclcpp;`
Just for symmetry

**Review by themarpe — CHANGES_REQUESTED** [review]

File: depthai_bridge/src/DisparityConverter.cpp (hunk 4 above)
Comment (saching13) [reply depthai_bridge/src/DisparityConverter.cpp:45]: rosOrigin is defined only in `BridgePublisher` because thats where it was most needed.

Comment (themarpe) [reply depthai_bridge/src/DisparityConverter.cpp:46]: So disparity converter is ros2 only?

Comment (saching13) [reply depthai_bridge/src/DisparityConverter.cpp:46]: No. the common function under ros and rclcpp was `ok()` which is called in while function in Bridge Publisher thats the only point where I needed to have a way to address it. 
In rest of the classes I only need Time which is `ros::Time` or `rclcpp.Clock` so there was no need to define `rosOrigin`

Comment (themarpe) [reply depthai_bridge/src/DisparityConverter.cpp:47]: Sounds good

**Pull request merged**